client/lib/importer/actions.js

```diff
@@ -10,6 +10,14 @@ const wpcom = require( 'lib/wp' ).undocumented();
 import { actionTypes } from './constants';
 import { fromApi, toApi } from './common';
 
+const apiSuccess = (...args) => {
+	Dispatcher.handleViewAction( {
+		type: actionTypes.API_SUCCESS
+	} );
+
+	return args;
```

Comment:
`apiSuccess` returns its arguments in an array - is that intentional? In the promise chain below, that has the side effect of wrapping the response from `fetchImporterState` in an array.